modules/@apostrophecms/permission/ui/apos/components/AposInputRole.vue

```diff
@@ -109,8 +114,13 @@ export default {
       // Allows expression of non-string values
       this.next = this.choices.find(choice => choice.value === JSON.parse(value)).value;
     },
-    async getTypes() {
-      return (await apos.http.get(`${apos.permission.action}/grid`, {})).types;
+    async getTypes(role) {
```

Comment:
It's really `getPermissions`, right? Or `getPermissionSets`? `getTypes` seems like it's populating `choices`.